doc/nrf/device_guides/fem/fem_output_power.rst

```diff
@@ -0,0 +1,20 @@
+.. _ug_radio_fem_sw_support_mpsl_fem_output:
+
+Setting the FEM output power
+############################
+
+The ``tx_gain_db`` property in devicetree provides the FEM gain value to use with the simple GPIO FEM implementation.
+The property must represent the real gain of the FEM.
+This implementation does not support controlling the gain value during runtime.
+
+nRF21540 implementations have the gain set to ``10`` by default.
+You can set a different gain value to use through the :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB`  option, but it has to match the value of one of the POUTA (:kconfig:option:`CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB_POUTA` ) or POUTB (:kconfig:option:`CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB_POUTB`) gains.
+
+.. caution::
+   :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB_POUTA` and :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB_POUTB` are by default set to ``20`` and ``10`` and these are factory-precalibrated gain values.
+   Do not change these values, unless POUTA and POUTB were calibrated to different values on specific request.
+
+To enable runtime control of the gain, set the :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_RUNTIME_PA_GAIN_CONTROL` to ``y``.
+This option makes the gain of the FEM to be adjusted dynamically during runtime, depending on the power requested by the protocol driver for each transmission.
+For the nRF21540 GPIO implementation, you must enable the **MODE** pin in devicetree.
+For the nRF21540 GPIO+SPI implementation, no additional configuration is needed as the gain setting is transmitted over the SPI bus to the nRF21540.
```

Comment:
Related to previous note. This content is related to nRF21540 only, does not apply to Simple GPIO.